### ISSUE TITLE
The password strength circle is blocked by password vault

### ISSUE THREAD

**Issue by karchbold** [jjcm/soci#85]
![image](https://user-images.githubusercontent.com/49083233/116836167-dc429900-ac08-11eb-8d8a-5a17a1237522.png)

**Comment by jjcm** [comment]
Wontfix as I can't control this. At the very least the bottom bar matches the entropy check circle.
